src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      this.result = 0;
+
+      return this;
+    },
+
+    add(value) {
+      return (this.result += value);
+    },
+
+    subtract(value) {
+      return (this.result -= value);
+    },
+
+    multiply(value) {
+      return (this.result *= value);
```

Comment:
[CODE STYLE] - The multiply method should also follow the same pattern as add and subtract, performing the operation and then returning the calculator object.